scripts/build-dist-debug.sh

```diff
@@ -18,7 +18,7 @@ umask 0022 # Ensure permissions are correct (0755 for dirs, 0644 for files)
 tar -xzf pack.tgz --strip 1
 rm -rf pack.tgz
 # Change this to "yarn install --production" once #1115 is fixed
```

Comment:
This comment doesn't really make sense any more, right? It should probably be removed 😛